ccmlib/scylla_repository.py

```diff
@@ -49,10 +49,10 @@ def setup(version, verbose=True):
         download_version(version, verbose=verbose, url=url, target_dir=os.path.join(
             tmp_download, 'scylla-core-package'))
 
-        url = os.environ.get('SCYLLA_JAVA_TOOLS_PACKAGE', os.path.join(
+        url = os.environ.get('SCYLLA_TOOLS_JAVA_PACKAGE', os.path.join(
```

Comment:
please don't change those names, it's use in multiple other repos already.
Now this PR can run in BYO...